src/app/config/config.local.js

```diff
@@ -2,7 +2,7 @@
   window.__env = window.__env || {};
 
   // API url
-  window.__env.apiUrl = 'http://api.hid.vm/api/v2/';
+  window.__env.apiUrl = 'http://api.hid.vm:3000/api/v2/';
```

Comment:
Since I have to keep reapplying this as I checkout new branches, I will merge this into dev to smooth over development.